pages/[user]/[type].tsx

```diff
@@ -184,6 +189,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       date: dateParam,
       eventType: eventTypeObject,
       workingHours,
+      ...(await serverSideTranslations(locale, ["availability-page"])),
```

Comment:
Is this required on every page?

Reply:
According to `next-i18next` docs:
`The serverSideTranslations HOC is primarily responsible for passing translations and configuration options into pages, as props – you need to add it to any page that has translations.`